src/main/java/com/xpanxion/java/springboot/da1/demo/service/student1/GymService1.java

```diff
@@ -25,9 +34,39 @@
     @Autowired
     private GymMemberRepository1 gymMemberRepository1;
 
+    @Autowired
+    private WorkoutHistoryRepository1 workoutHistoryRepository1;
+
     @Autowired
     private TimestampsRepository1 timestampsRepository1;
 
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    private final String SELECT_MEMBER_WORKOUT_HISTORY = "select * from workout_history1 where member_id = ?";
```

Comment:
Just curious, what was the rationale for using a `JdbcTemplate` here?

Reply:
I wasn't sure how else to retrieve the data from the data base without a query. Is there no need for this approach?